vnext/JSI/Shared/ChakraRuntimeArgs.h

```diff
@@ -2,49 +2,71 @@
 // Licensed under the MIT License.
 
 #pragma once
-#include <jsi/jsi.h>
-#include "ScriptStore.h"
+
+#include "PreparedJavaScriptStore.h"
+
+#include <cstddef>
+#include <memory>
+#include <optional>
+#include <utility>
 
 namespace facebook::react {
 class MemoryTracker;
-class MessageQueueThread;
-} // namespace facebook::react
+}
 
 namespace Microsoft::JSI {
 
-enum class LogLevel {
-  Trace = 0,
-  Info = 1,
-  Warning = 2,
-  Error = 3,
-  Fatal = 4,
-};
-
-using Logger = std::function<void(const char *message, LogLevel logLevel)>;
-
 struct ChakraRuntimeArgs {
-  bool enableJITCompilation{true};
+ public:
```

Comment:
>public [](start = 1, length = 6)

nit: public is implied for struct

Reply:
I like to explicitly state the accessibly of members. If people feel strongly about this I can remove it.

---
In reply to: [345457882](https://github.com/microsoft/react-native-windows/pull/3650#discussion_r345457882) [](ancestors = 345457882)